server/data/data-jmap/src/test/java/org/apache/james/jmap/memory/vacation/MemoryVacationRepositoryTest.java

```diff
@@ -0,0 +1,77 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.memory.vacation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.james.jmap.api.vacation.AccountId;
+import org.apache.james.jmap.api.vacation.Vacation;
+import org.apache.james.jmap.api.vacation.VacationRepository;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MemoryVacationRepositoryTest {
+
+    public static final AccountId ACCOUNT_ID = AccountId.create("identifier");
+    public static final Vacation VACATION_ID_1 = Vacation.builder().enabled(true).build();
+    public static final Vacation VACATION_ID_2 = Vacation.builder().enabled(true).build();
```

Comment:
Good catch. I removed _ID . Later commit (cassandra one makes the value differents while generalizing the tests.